elmslie-android/src/main/java/vivid/money/elmslie/android/screen/ElmScreen.kt

```diff
@@ -69,7 +69,8 @@ class ElmScreen<Event : Any, Effect : Any, State : Any>(
         screenLifecycle.addObserver(lifecycleObserver)
     }
 
-    private fun isAllowedToRunMvi() = !isAfterProcessDeath || activityProvider() !is StopElmOnProcessDeath
+    private fun isAllowedToRunMvi() =
+        delegate.isAllowedToRunMvi && (!isAfterProcessDeath || activityProvider() !is StopElmOnProcessDeath)
```

Comment:
If delegate.isAllowedToRunMvi = false then isAllowedToRunMvi is always false then initEvent in lifecycleObserver.onCreate will never be triggered.

So this solution doesn't work.

Reply:
That's exactly what I want to achieve. I set delegate.isAllowedToRunMvi to false after restoring from memory and redirect to another screen right after it.